### ISSUE TITLE
Can not link using msvc

### ISSUE THREAD

**Issue by fluxxu** [sgrif/pq-sys#3]
Hi,

I tried to install diesel_cli on my Windows 10
It says

```
note: LINK : fatal error LNK1181: cannot open input file 'pq.lib'
```

There is a `libpq.lib` in the pqsql/lib folder, if I rename it to `pq.lib`, diesel_cli can be installed successfully...
Maybe this is a cargo's issue?

Thanks. 

https://github.com/diesel-rs/diesel/issues/487

**Comment by tyoc213** [comment]
same here? was trying to use diesel on Win 10... but got this issue.

Also exactly what postgree do I need to install @fluxxu ?

**Comment by fluxxu** [comment]
@tyoc213 

For Postgres server, I use Docker because I don't want Postgres to create user on my computer.
For client library to link with diesel, I used EnterpriseDB zip archive here:
https://www.enterprisedb.com/download-postgresql-binaries
Assume you extract it to X:\pgsql
You need to rename or copy X:\pgsql\lib\libpq.lib to X:\pgsql\lib\pq.lib otherwise cargo can not find it.
Don't forget to set environment variable PQ_LIB_DIR to X:\pgsql\lib

Related issue:
https://github.com/diesel-rs/diesel/issues/487

😄

**Comment by rlepidi** [comment]
This still isn't working for me. I know I have the updated version because it says `LNK1181: cannot open input file 'libpq.lib'`. I have my `PG_LIB_DIR` env var set to `C:\Program Files\PostgreSQL\9.4\lib` and can confirm `libpq.lib` lives there. Is there something else I can try?

**Comment by sgrif** [comment]
`PG_LIB_DIR` doesn't do anything. As per the readme, you need to do on of:

- have `LIB_PQ_DIR` is set and pointing to the right directory
- have `pg_config.exe` on your `PATH`
- have `pkg-config` on your `PATH` and configured to be able to find libpq.

**Comment by rlepidi** [comment]
Sorry, that was a typo. I actually have it set as `PQ_LIB_DIR`. (readme has it wrong, see my PR)

I figured it out. I was trying all these things and just doing a `cargo build` on my project, which never ran `build.rs` again since `pq-sys` already built successfully (it just failed to link). Once I did a `cargo clean` and it reran build.rs, it picked up my env var and linked successfully.

**Comment by ysinsane** [comment]
I encounted this issue too. Firstly, I get this 
`LNK1181: cannot open input file 'libpq.lib'`
Then I installed postgrel server.(I don't really understand I need to install a server to get this lib. Why libpq.DLL is not enough.)
Now I get 
`note: LINK : fatal error LNK1181: cannot open input file 'pq.lib'`
So I copy libpq.lib and named it as 'pq.lib'. Then I run cargo install(I did not run cargo clean) ... everything works!
